PathfinderHonorManager/Controllers/PathfinderHonorsController.cs

```diff
@@ -64,10 +64,8 @@ public async Task<IActionResult> PostAsync(Guid pathfinderId, [FromBody] Incomin
             var pathfinderHonor = await _PathfinderHonorService.AddAsync(pathfinderId, newPathfinderHonor, token);
 
             return CreatedAtRoute(
-                GetByIdAsync(pathfinderHonor.PathfinderID, pathfinderHonor.PathfinderHonorID, token),
```

Comment:
`pathfinderHonor` appears to contain everything we need in this case.
GetByIdAsync returns a Task which gets handed directly to the routing infra unfortunately resulting in a route that isn't correct (we need the raw DTO).